cycode/cli/code_scanner.py

```diff
@@ -1099,6 +1112,21 @@ def _is_file_extension_supported(scan_type: str, filename: str) -> bool:
     return not filename.endswith(consts.SECRET_SCAN_FILE_EXTENSIONS_TO_IGNORE)
 
 
+def _is_iac(scan_type: str) -> bool:
+    return scan_type == consts.INFRA_CONFIGURATION_SCAN_TYPE
+
+
+def _is_tfplan_json_file(file: str, content: str) -> bool:
+    if not file.endswith('.json'):
+        return False
+    try:
+        tf_plan = json.loads(content)
+        return tf_plan.get('resource_changes')
```

Comment:
maybe check `terraform version` key exists as well?

Reply:
@MichalBor My concern is that if terraform_version won't be exists we won't scan the plan even if resource_changes exists.. from other hand, if teffaform_version is always exists it will do the condition stronger which is good. need to think about it a little bit more.

Reply:
Need to see if terraform version affects the plan structure (and then to the convert logic)

Reply:
Terraform 0.12 and later is supported